### ISSUE TITLE
Express sortability (even if not currently sorted) of columns

### ISSUE THREAD

**Issue by michael-n-cooper** [w3c/aria#724]
Migrated from [ISSUE-54](https://www.w3.org/WAI/ARIA/track/issues/524). Need a way to indicate that columns could be sorted. aria-sort="none" doesn't work because it's default value meaning all columns have that value implicitly. Might need to add an "undefined" value and make it the default, where one means not sortable and the other means sortable but not sorted.

**Comment by jnurthen** [comment]
Bulk closing stale issues - please reopen if necessary
